describe/EnumVal.ts

```diff
@@ -0,0 +1,20 @@
+export type EnumValDescription = {
+  /** the attribute `ord` of the element `EnumVal` */
+  ord?: number;
```

Comment:
Why does this have a question mark? Can it be undefined under some circumstances?

Reply:
Very good question @ca-d . I am really unsure about this myself. The reason is this: The attribute `ord` in `EnumVal` is required. The question though is how do we handle schema invalid files. XML is not enforcing the attribute to be set. I had several ideas here:

1. Return `undefined` for invalid `EnumVal` element. 
2. Return `NaN` for this specific case

Btw. that is a general question. How do we handle required attributes that incorrectly are missing.